WPMedia/engine-theme-sdk
# TMEDIA-231 : Add an optional setting to confine video heights in bloc…

## Description
Add support for video elements to resize based on vertical height

## Jira Ticket
- [TMEDIA-231](https://arcpublishing.atlassian.net/browse/TMEDIA-231)

## Acceptance Criteria
- When videos are within the article body, the video should be entirely visible within the viewport, like the example screenshot shows above.
- The ratio of the video should be kept the same, and the video should be centered horizontally within the article body.
- Text should not wrap around the video.
- Tests and documentation are updated as necessary

## Test Steps
- Checkout this branch git checkout TMEDIA-231-video-height-constraints
- Ensure `.env` is configured to use local `engine-theme-sdk`
- Run fusion repo with linked blocks npx fusion start -f -l @wpmedia/engine-theme-sdk,@wpmedia/article-body-block,@wpmedia/extra-large-promo-block,@wpmedia/large-promo-block,@wpmedia/lead-art-block,@wpmedia/video-player-block,@wpmedia/video-promo-block,@wpmedia/top-table-list-block

## Effect Of Changes
See: https://github.com/WPMedia/fusion-news-theme-blocks/pull/884

## Dependencies or Side Effects
n/a

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
